packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/RealmObjectHelper.kt

```diff
@@ -759,6 +761,141 @@ internal object RealmObjectHelper {
         }
     }
 
+    @Suppress("LongParameterList", "NestedBlockDepth", "LongMethod", "ComplexMethod")
+    internal fun assignValuesOnUnmanagedObject(
+        target: BaseRealmObject,
+        source: BaseRealmObject,
+        mediator: Mediator,
+        currentDepth: Int,
+        maxDepth: Int,
+        closeAfterCopy: Boolean,
+        cache: ManagedToUnmanagedObjectCache
+    ) {
+        val metadata: ClassMetadata = source.realmObjectReference!!.metadata
+        for (property in metadata.properties) {
+            val accessor = property.accessor
+                ?: sdkError("Typed object should always have an accessor")
+            if (property.isComputed) {
```

Comment:
We don't populate backlinks? we already have the reference to the targets...

Reply:
Mostly because we don't support unmanaged versions of `LinkingObjects` it will throw an exception normally. So it would feel weird if it was allowed for objects that had been copied.